WikiWatershed/model-my-watershed
# Support Low-Density Open Space SRAT Results

## Overview

The SRAT API was already sending back a `tiledrain` set of results which we were not using. They have, for the time being, switched to sending Low-Density Open Space results in that key, so we show it as such.

Connects #2954

### Demo

![image](https://user-images.githubusercontent.com/1430060/47379981-66b9f180-d6ca-11e8-9566-e65b9ff9444e.png)

### Notes

I'm going to manually apply this patch to staging and let BME know so he can check the numbers and suggest any changes if necessary, which will be incorporated into this PR before merging.

## Testing Instructions

* Ensure the `/etc/mmw.d/env/MMW_SRAT_CATCHMENT_API_URL` in `app` and `worker` points to https://rychrnjrs6.execute-api.us-east-1.amazonaws.com/dev/SratRunModel_DEV
* Ensure the `etc/mmw.d/env/MMW_SRAT_CATCHMENT_API_KEY` in `app` and `worker` is filled out (the key is in LastPass)
* Check out this branch and restart Celery

    ```
    $ vagrant ssh worker -c 'sudo service celeryd restart'
    ```

* Go to [:8000](http://localhost:8000) and create a new MapShed project. View subbasin results.
* Ensure you see "Low-Density Open Space" in the list of results
